packages/math-input/less/main.less

```diff
@@ -1,2 +1,2 @@
 @import "./overrides.less";
-@import (inline) "../../../node_modules/mathquill/build/mathquill.css";
+@import (inline) "../node_modules/mathquill/build/mathquill.css";
```

Comment:
pnpm installs direct package dependencies into the package's local `node_modules` folder, so these relative paths now make more sense.